client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.scss

```diff
@@ -15,6 +15,17 @@
 	margin: 0;
 	list-style-type: none;
 	overflow: auto;
+	// Expected to be changed by using the theme setting "styles.color.background"
+	background: var(--studio-white);
+
+	> li {
+		// Expected to be changed by using the theme setting "styles.spacing.blockGap"
+		margin-block-start: 1.5rem;
```

Comment:
This should only apply to the header and footer as we don't have the gap between section patterns

Reply:
You are right, because the sections are in the same group in the template.

Reply:
I have updated the CSS. What do you think about using a class?

Reply:
I'm okay with the current way 🙂